dataset_configs/armenian/audio_books/config.yaml

```diff
@@ -0,0 +1,107 @@
+documentation: |
+  Audio books
+  ######
+
+  This config can be used as example to process audiobooks in Armenian language and prepare 
+  dataset in the NeMo format.
+
+  This config performs the following data processing.
+
+  1. Create initial manifest by collecling all avalible files with mp3 expention in raw_data_dir folder.
+     
+  2. Convert mp3 into wav format using the Ffmpeg suite, with a downsampling to a 16000 Hz sample rate 
+  and a unification of all audio channels into a mono track.
+  3. Count duration for audio files in seconds and save it into duration field.
+  4. Filter out broken files with duration shorter than 0 seconds.
+     You can directly change the config file to control this.
+  5. Predict transcription using large-v2 Whisper ASR model into text field.
+  6. Predict transcription using distil-whisper/distil-large-v2 transformers ASR model into pred_text field.
+  7. Drops everything with non-armenean characters.
```

Comment:
Please check that the built docs look OK. Currently this numbered list does not look correct. You need to add spaces between each list entry, and probably get rid of the indent before "You can directly change the config file to control this"

Reply:
done